src/panels/lovelace/common/handle-action.ts

```diff
@@ -91,7 +91,10 @@ export const handleAction = (
         return;
       }
       const [domain, service] = actionConfig.service.split(".", 2);
-      hass.callService(domain, service, actionConfig.service_data);
+      hass.callService(domain, service, {
+        entity_id: config.entity,
```

Comment:
What if the service call doesn't accept `entity_id`?

Reply:
💩